Src/AutoFixture/RandomNumericSequenceGenerator.cs

```diff
@@ -53,7 +51,17 @@ public RandomNumericSequenceGenerator(params long[] limits)
 
             if (limits.Length < 2)
             {
-                throw new ArgumentException("The limit must be a sequence of two or more integers.");
+                throw new ArgumentException("limits must be two or more Int64.", "limits");

```

Comment:
This is a bit confusing to me. –  Why is `Int64` written in the exception message?

Wouldn't it be easier by re-using the sentence provided in the XML documentation for the parameter?

>A sequence of at least two ascending numbers.

(Or similar.)

---

Also, I'd be nice if we'd use correct [grammar, spelling, and punctuation](http://blog.ploeh.dk/2014/12/23/exception-messages-are-for-programmers/), by starting with a capital letter.